### PR TITLE
Cache the supports calculated during CCSpan

### PR DESCRIPTION
**Pull request by FWDekker**
Caches the supports calculated during CCSpan.

Review only commit `Cache the supports calculated during CCSpan`.

### CODE REVIEW

File: modules/mining-pipeline/ccspan-pattern-detector/src/main/kotlin/org/cafejojo/schaapi/miningpipeline/patterndetector/ccspan/CCSpan.kt [hunk 1]
```diff
@@ -87,10 +94,12 @@ internal class CCSpan<N : Node>(
         }
     }
 
-    private fun calculateSupport(sequence: List<N>) =
-        sequences
+    private fun calculateSupport(sequence: List<N>): Int {
```
[the remark this answers is not in the document]
Comment (FWDekker) [reply modules/mining-pipeline/ccspan-pattern-detector/src/main/kotlin/org/cafejojo/schaapi/miningpipeline/patterndetector/ccspan/CCSpan.kt]: Detekt warned me about it as well

**Comment by FWDekker** [comment]
@JSAbrahams It is very fast. The fastest pipeline ever. So fast.

File: modules/mining-pipeline/ccspan-pattern-detector/src/main/kotlin/org/cafejojo/schaapi/miningpipeline/patterndetector/ccspan/CCSpan.kt [hunk 2]
```diff
@@ -77,7 +84,7 @@ internal class CCSpan<N : Node>(
 
     private fun identifyNonClosedSequences() {
         sequencesOfCurrentLength.forEach { (sequence, sequenceSupport, _) ->
-            sequencesOfPreviousLength.parallelStream()
+            sequencesOfPreviousLength
```
Comment (gandreadis) [inline modules/mining-pipeline/ccspan-pattern-detector/src/main/kotlin/org/cafejojo/schaapi/miningpipeline/patterndetector/ccspan/CCSpan.kt]: Why did you remove parallelStream()?
